### DISCUSSION TITLE
How to Retarget/Create Control Rig

### DISCUSSION THREAD

**Discussion by ve110cet** [Sixze/ALS-Refactored#203] [General]
I've got my character retargeted to the B_Als_Character, but there are a few issues including IK not working.

Going through it looks like having a proper Control Rig will solve some (maybe all) of these issues.  How do I retarget the Control Rig to my new character (all the bone names/chains are different) or how do I create a new one that will work with the retargeted animation blueprint?

**Comment by woodencase01** [comment]
I still haven't found how to make it look perfect yet (I have a bit of twist in the foot), but here is what I did:
- Retarget the CR_Als COntrol Rig
- Assign it in the main Animation BluePrint
- In "RefreshFootIK" from the CR, get both the primary aix and secondary axis values corresponding to your skeleton
 
![image](https://user-images.githubusercontent.com/4787041/229287351-cc130830-d414-4894-9b19-38e001fdecb0.png)
